installer/docker_setup.py

```
@@ -211,35 +219,86 @@ def create_docker_context(self):
+            
+            os.environ["DOCKER_HOST"] = f"tcp://{local_ip}:{docker_port}"
+            os.environ["DOCKER_TLS_VERIFY"] = "1"
+            os.environ["DOCKER_CERT_PATH"] = str(self.docker_certs_dir)
+            print("\nChecking Docker contexts...")
+            context_result = subprocess.run(
+                ["docker", "context", "ls"],
+                capture_output=True,
+                text=True
+            )
+            
+            print(f"\checking systemd")
```

Ruff (0.11.9)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)